src/game/emutest.h

```diff
@@ -5,22 +5,19 @@
 
 enum Emulator {
     EMU_WIIVC = 0x0001,
-    EMU_PROJECT64_ANY = 0x001E,
-        EMU_PROJECT64_1_OR_2 = 0x0006, // PJ64 1.6 or similar (up to 2.3)
-        EMU_PROJECT64_3 = 0x0008, // PJ64 3.X
-        EMU_PROJECT64_4 = 0x0010, // PJ64 4.0 or later
-    EMU_MUPEN_BASED = 0x0060, // mupen64plus or pre-2.12 paralleln64, but NOT simple64 or new paralleln64
-        EMU_MUPEN_OLD = 0x0020, // Nemu64 and pre-2.12 paralleln64 will also get detected as this
-        EMU_MUPEN64PLUS_NEXT = 0x040,
-    EMU_CEN64 = 0x0080,
-    EMU_SIMPLE64 = 0x0100,
-    EMU_PARALLELN64 = 0x0200, // ParallelN64 2.12 or later
-    EMU_ARES = 0x0400,
-    EMU_CONSOLE = 0x0800
+    EMU_PROJECT64_ANY = 0x006,
+        EMU_PROJECT64_1_OR_2 = 0x0002, // PJ64 1.6 to 2.3
+        EMU_PROJECT64_3 = 0x0004, // PJ64 2.4 to 3.0
+    EMU_MUPEN_BASED = 0x0018,
```

Comment:
We don't really need to distinguish between new and old mupen.
My proposed groupings:
```
EMU_CONSOLE,
EMU_WIIC,
EMU_INSTANT_INPUT
    EMU_PARALLEL_LAUNCHER
    EMU_PROJECT_64_1
    EMU_OTHER_INSTANT_INPUT
EMU_OTHER
```